packages/contracts/test/utils/abi-encoder-v2.js.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"abi-encoder-v2.js","sourceRoot":"","sources":["abi-encoder-v2.ts"],"names":[],"mappings":";;AAAA,wCAA+C;AAE/C,SAAgB,oBAAoB,CAAC,gBAAwB;IAC3D,MAAM,WAAW,GAAa,EAAE,CAAC;IACjC,MAAM,KAAK,GAAG,gBAAgB,CAAC,KAAK,CAAC,GAAG,CAAC,CAAC;IAC1C,KAAK,MAAM,IAAI,IAAI,KAAK,EAAE;QACxB,MAAM,UAAU,GAAG,IAAI,CAAC,IAAI,EAAE,CAAC;QAC/B,IAAI,UAAU,CAAC,MAAM,KAAK,CAAC,EAAE;YAC3B,SAAS;SACV;QACD,MAAM,KAAK,GAAG,UAAU,CAAC,KAAK,CAAC,GAAG,CAAC,CAAC;QACpC,IAAI,KAAK,CAAC,MAAM,KAAK,CAAC,EAAE;YACtB,MAAM,IAAI,KAAK,CACb,wDAAwD,UAAU,GAAG,CACtE,CAAC;SACH;QACD,WAAW,CAAC,IAAI,CAAC,UAAU,CAAC,CAAC;KAC9B;IACD,OAAO,SAAS,WAAW,CAAC,IAAI,CAAC,IAAI,CAAC,GAAG,CAAC;AAC5C,CAAC;AAjBD,oDAiBC;AAED,SAAgB,YAAY,CAAC,QAAgB,EAAE,MAAc;IAC3D,OAAO,uBAAe,CAAC,MAAM,CAAC,CAAC,QAAQ,CAAC,EAAE,CAAC,MAAM,CAAC,CAAC,CAAC;AACtD,CAAC;AAFD,oCAEC"}
```

Comment:
is this `.map` file meant to be committed?

Reply:
no